src/google/Line.js

```diff
@@ -26,76 +35,71 @@
 
     Line.prototype.publish("smoothLines", true, "boolean", "Causes chart data lines to draw smoothly");
```

Comment:
yes, there is more changes coming down the line though ... will just wait for the next PR tonight before the weekend